cabal-install/Distribution/Client/CmdInstall.hs

```diff
@@ -628,6 +464,164 @@ installAction ( configFlags, configExFlags, installFlags
                   haddockFlags testFlags benchmarkFlags
     globalConfigFlag = projectConfigConfigFile (projectConfigShared cliConfig)
 
+
+getClientInstallFlags :: Verbosity -> GlobalFlags -> ClientInstallFlags -> IO ClientInstallFlags
```

Comment:
The name should hint that this is used for `ignore-project` cases.

Reply:
Hmm, I'm not familiar with `ignore-project` cases or what they imply so can't really come up with a better name, let me know if you have a suggestion.